src/main/java/government/model/User.java

```diff
@@ -21,6 +23,9 @@
     @NotNull
     private String password;
 
+    @OneToMany(mappedBy = "user")
+    private List<Ownership> ownerships;
```

Comment:
Is `List` the most basic interface you need for this class? Is `Collection` not sufficient?